pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/documentation/CommentContentRule.java

```diff
@@ -143,11 +143,16 @@ public Object visit(ASTCompilationUnit cUnit, Object data) {
         return super.visit(cUnit, data);
     }
 
-    public boolean hasDissallowedTerms() {
+    public boolean hasDisallowedTerms() {
```

Comment:
```suggestion
    private boolean hasDisallowedTerms() {
```

This is part of the internals of this rule. So, the old method can be deprecated, but this new methods does not need to be public

Reply:
@oowekyala done.